### PR TITLE
Multiple providers support

### PR DESCRIPTION
**Pull request by lucas-gauer**
- [x] Expose window.ethereum
- [x] Update dapp store/controller to handle different chains
- [x] Change network when pali network differs from dapp network
- [x] Change the account when pali account differs from the connected account

### CODE REVIEW

File: source/utils/network.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import store from 'state/store';
+
+export const isActiveNetwork = (chain: string, chainId: number) => {
+  const { activeNetwork } = store.getState().vault;
+  const isSysCore = activeNetwork.url.includes('blockbook');
```
Comment (amandagonsalves) [inline source/utils/network.ts:5]: check for chain as well, we can't tell if it is a syscoin core network or evm just by the url

**Review by amandagonsalves — CHANGES_REQUESTED** [review]
- pali connected sites not showing when there is a dapp connected
- disconnect from pali not working

File: source/utils/network.ts (hunk 1 above)
Comment (lucas-gauer) [reply source/utils/network.ts:5]: This is still a terrible way of doing it. I just copied the same solution that i saw somewhere else. Will update

**Comment by amandagonsalves** [comment]
> > * pali connected sites not showing when there is a dapp connected
> > * disconnect from pali not working
> 
> Neither is related to multiple providers support. Disconnect seems to work fine, will test again.

is related to inject anyway, if there is something missing i'll point it out just to make sure we don't forget to implement anything

**Review by amandagonsalves — APPROVED** [review]

**Pull request merged**